float32 x
float32 y
string is_centered
string pickup_state